parl/utils/env_utils.py

```diff
@@ -0,0 +1,98 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym
+from parl.utils import logger
+from parl.remote.remote_decorator import remote_class
+from gym.spaces import Box, Discrete
+
+__all__ = ['RemoteGymEnv']
+
+
+@remote_class
+class RemoteGymEnv(object):
+    """
+
+    Example:
+    .. code-block:: python
+        import parl
+        from parl.utils import RemoteEnv
+
+        parl.connect('localhost')
+        env = RemoteGymEnv(env_name='HalfCheetah-v1')
+
+    Attributes:
+        env_name: Mujoco gym environment name
+
+    Public Functions: the same as gym (mainly action_space, observation_space, reset, step, seed ...)
+
+    Note:
+        ``RemoteGymEnv`` defines a remote environment wrapper for machines that do not support mujoco,
```

Comment:
defines a remote environment wrapper for running the environment remotely and enable large-scale parallel collection of environmental data

Reply:
done